internal/indexer/testdata/TestIndexer_documentation/testdata/cmd.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
What is this file?

Reply:
No idea, it got autogenerated by the tests, so I thought it was supposed to be committed haha